<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomNumber</class>
 <widget class="QWidget" name="RoomNumber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_roomnumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>房间号列表</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_roomnumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>311</width>
     <height>571</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_createroom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>创建直播间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_joinroom">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>660</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加入直播间</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
